### PR TITLE
Clean up exported credentials when the workflow finishes

### PR DESCRIPTION
**Pull request by sethvargo**
Fixes https://github.com/google-github-actions/auth/issues/66

### CODE REVIEW

File: src/post.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+/**
+ * Executes the post action, documented inline.
+ */
+export async function run(): Promise<void> {
+  try {
+    const cleanupCredentials: boolean = getBooleanInput('cleanup_credentials');
+    if (cleanupCredentials) {
+      return;
+    }
```
Comment (bharathkkb) [inline src/post.ts:14]: Shouldn't this be `if !cleanupCredentials` return early?

File: src/utils.ts [hunk 2]
```diff
@@ -25,6 +27,23 @@ export async function writeSecureFile(outputDir: string, data: string): Promise<
...
+  // Look up the credentials path, if one exists. Note that we only check the
+  // environment variable set by our action, since we don't want to
+  // accidentially clean up if someone set GOOGLE_APPLICATION_CREDENTIALS or
+  // another environment variable manually.
+  const credentialsPath = process.env['GOOGLE_GHA_CREDS_PATH'];
+  if (!credentialsPath) {
+    return;
+  }
+
+  // Delete the file.
```
Comment (bharathkkb) [inline src/utils.ts:43]: Should we add some info logging after this, saying `deleted ${credentialsPath}`?

**Comment by sethvargo** [comment]
Hmm - so it actually looks like the credentials file is removed automatically

**Comment by sethvargo** [comment]
auth uses `RUNNER_TEMP`, so it's automatically cleaned up. `setup-gcloud` uses `GITHUB_WORKSPACE` which is why its not being cleaned up.

**Review by bharathkkb — APPROVED** [review]

**Pull request merged**